x-pack/test_serverless/api_integration/test_suites/common/index.ts

```diff
@@ -14,5 +14,6 @@ export default function ({ loadTestFile }: FtrProviderContext) {
     loadTestFile(require.resolve('./security_response_headers'));
     loadTestFile(require.resolve('./rollups'));
     loadTestFile(require.resolve('./index_management'));
+    loadTestFile(require.resolve('./alerting/rules'));
```

Comment:
**nit:** to follow the pattern of other folders, can we add an `index.ts` file within `alerting` folder and require the folder instead? The `index.ts` file can require `/rules` and in the future other alerting related tests.
```suggestion
    loadTestFile(require.resolve('./alerting'));
```

Reply:
Sure! Resolved in this commit [f0df4c5](https://github.com/elastic/kibana/pull/163753/commits/f0df4c5af9f736ace417701b63686380388c4f66)